testutil/cli/cmd.go

```diff
@@ -0,0 +1,27 @@
+package cli
+
+import (
+	"context"
+
+	"github.com/spf13/cobra"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/testutil"
+)
+
+// ExecTestCLICmd builds the client context, mocks the output and executes the command.
+func ExecTestCLICmd(clientCtx client.Context, cmd *cobra.Command, extraArgs []string) ([]byte, error) {
```

Comment:
Let's return `out` here instead of `[]byte`. Otherwise, ACK 🎉

Reply:
Done!